COMPILATION.md

```diff
@@ -15,7 +15,10 @@ Keep in mind using the pre-built packages when available.
 * libcurl
 * libxml2
 * openssl
-* /etc/mime.types (the package providing depends on the OS)
+* mime.types (the package providing depends on the OS)
+	* /etc/mime.types is default for not macOS
+	* /etc/apache2/mime.types is default for macOS
+	* If there is a file in a path other than the default, specify it with the mime option.
```

Comment:
Shouldn't it be
>	* If there is a file in a path other than the default, specify it with the `--mime` option.

?

Reply:
Well right, not `--mime` but just `mime` :-)

Reply:
Is below description better than it?
- Alternatively, the mime.types file path can be specified with the `mime` option.

Reply:
Yes, but on a second look, I'd use `/etc/mime.types`, `/etc/apache2/mime.types` and `mime.types` in all cases.

Reply:
Since the search path and priority were changed, the code was squashed again and the description was almost changed.
Please check the new wording.